examples/lusid/orders/blocks/3-create-custom-property-in-block-domain.sql

```diff
@@ -0,0 +1,33 @@
+-- set property variables
```

Comment:
Description: Create a custom property to store the contingent order id

Reply:
updated